### ISSUE TITLE
Dota Training Time and test setting

### ISSUE THREAD

**Issue by byfate** [yijingru/BBAVectors-Oriented-Object-Detection#64]
Question About Dota Training time:
I am training Dota dataset trainval now ,it has **69337** total (train+val scale 1.0+0.5),and it needs training for 40 hours when using I use batch size=20  one epoch,May you tell me how long will it take when you training model using 4 1080Ti GPUS 

Question About Dota testing:
when generate dota testing image ,should I only use scale 1.0 or use both 1.0 and 0.5.  

And by the way, How to pick the best trained model to get the evalute result for  DOTA  online server?
thank you.Waiting for replying .Appreciated.

**Comment by yijingru** [comment]
1. Training time: my condition would be different as I shared the system I/O with other students. But it would take less than 3-5 days when training with 4 1080 Ti GPUs. The RTX 6000 would take about 2 days.
2. Both 1.0 and 0.5 scales are used for the testing set.
3. If you have enough resources, I think it would better to set aside a validation set from the trainval and monitor the loss/accuracy to get the best epoch. And then retrain the whole dataset and pick the weights at that epoch.

**Comment by byfate** [comment]
thank you

**Comment by byfate** [comment]
> 1. Training time: my condition would be different as I shared the system I/O with other students. But it would take less than 3-5 days when training with 4 1080 Ti GPUs. The RTX 6000 would take about 2 days.
> 2. Both 1.0 and 0.5 scales are used for the testing set.
> 3. If you have enough resources, I think it would better to set aside a validation set from the trainval and monitor the loss/accuracy to get the best epoch. And then retrain the whole dataset and pick the weights at that epoch.

thank you
